Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -1925,21 +1927,12 @@ - (void)hideUI
 #if PLATFORM(VISION) && ENABLE(QUICKLOOK_FULLSCREEN)
 
 @implementation WKFullScreenWindowController (WKSPreviewWindowControllerDelegate)
-- (void)previewWindowControllerDidClose
-{
-    [self requestExitFullScreen];
-}
-@end
-
-@implementation WKFullScreenWindowController (QLPreviewItemDataProvider)
-- (NSData *)provideDataForItem:(QLItem *)item
+- (void)previewWindowControllerDidClose:(WKSPreviewWindowController *)previewWindowController
```

Comment:
Nit: for consistency with the declaration in WKSPreviewWindowController.h you should use `id` for the type of `previewWindowController`